static/js/components/PersonalForm.js

```diff
@@ -39,6 +39,37 @@ export default class PersonalForm extends ProfileFormFields {
     ui:                     UIState,
   };
 
+  renderRomanizedFields = () => {
+    const { profile } =  this.props;
+    const english = /^[A-Za-z0-9]*$/;
```

Comment:
right now i am doing same, if name is not english then i am showing input box to add latin char names